x-pack/plugins/security_solution/server/index.ts

```diff
@@ -47,6 +47,7 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
       `xpack.securitySolution.${SIGNALS_INDEX_KEY}`,
       { level: 'critical' }
     ),
+    unused('ruleExecutionLog.underlyingClient', { level: 'warning' }),
```

Comment:
Looks good! 👍 

<p align="center">
  <img width="700" src="https://user-images.githubusercontent.com/2946766/150236150-8e91b038-bc21-47b4-86e6-34e751b73502.png" />
</p>

Do you happen to know how long these need to stick around? I think it used to be till the next major, but not sure now with make-it-minor?